### PR TITLE
Fix dev tools to work with v2

### PR DESCRIPTION
**Pull request by sterowney**

### CODE REVIEW

File: register.js [hunk 1]
```diff
@@ -1,24 +1,22 @@
...
-      }
-    ]
-  });
-}
+// Register package as ReactionCommerce package
+Reaction.registerPackage({
+  label: "Reaction Devtools",
+  name: "reaction-devtools",
+  icon: "fa fa-code",
+  meta: {
```
Comment (zenweasel) [inline register.js]: I know you copied this from the original but plugins have a top-level attribute called `version` now. Can we set this and make it `1.0.0` for this post 2.0 version?

Comment (sterowney) [reply register.js]: Sure, I added it as a top-level attribute and also bumped the meta-data one. Is that what you were expecting?

**Comment by loan-laux** [comment]
Would be great to have this merged. This tool is very useful to anyone who isn't working with a database dump. Any chance of merging it soon @zenweasel?

**Comment by loan-laux** [comment]
Thanks @zenweasel! 👍

**Pull request merged**